src/style.css

```diff
@@ -0,0 +1,117 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  color: #060b35;
+}
+
+.card__img {
+  width: 160px;
+  margin: 0 auto;
```

Comment:
Set correct height to the image (check it in figma), padding: 3px, width: 100% to pass tests

Reply:
Don't forget to add display: block to have an ability to set image size